Neos.Fusion/Classes/Service/RenderAttributesTrait.php

```diff
@@ -40,6 +40,9 @@ protected function renderAttributes(iterable $attributes, bool $allowEmpty = tru
                 // [""] => empty attribute
                 $joinedAttributeValue = '';
                 foreach ($attributeValue as $attributeValuePart) {
+                    if ($attributeValuePart instanceof \Stringable) {
```

Comment:
is `method_callable($attributeValuePart, '__toString')` more versatile?

Reply:
no i wasnt sure of the implicit magic :D so fine with me 🔥 thanks for the research